packages/frontapp/src/app/components/BookCardExtended/index.spec.tsx

```diff
@@ -0,0 +1,72 @@
+import { render } from '../../../helpers/customRender';
+import BookCardExtended from './index';
+import { IMaterial } from '../../types';
+import { StatusTypes } from '@mimir/global-types';
+import { screen } from '@testing-library/react';
+
+describe('render BookCardExtended', () => {
+  it('correctly render with status is free', () => {
+    const mockItem: IMaterial | null = {
+      author: 'Ivan',
+      category: 'Horror',
+      title: 'Dracula',
+      created_at: '2022-09-21T08:29:24.192Z',
+      id: '1',
+      picture: '',
+      statuses: [
+        {
+          id: '1',
+          created_at: '2022-09-21T08:29:24.192Z',
+          status: StatusTypes.FREE,
+          person: {
+            id: '1',
+            username: 'Dmitry',
+          },
+        },
+      ],
+    };
```

Comment:
I have an idea for snapshots - it might make sense to keep snapshot API data in JSON somewhere near snapshots that use that data.